src/main/java/banana/Ui.java

```diff
@@ -0,0 +1,44 @@
+package banana;
+
+import java.util.Scanner;
+
+/**
+ * The Ui class handles
+ * the user inputs.
+ *
+ * @author: Ravi Ananya
+ **/
+
+public class Ui {
+
+    protected Scanner userInput;
```

Comment:
Should another name be used in naming this Scanner variable instead? Naming it **userInput** may cause confusion to readers, since the purpose of this variable is to read and obtain user input, and readers may misinterpret that this variable is used to store user input instead.